rakelib/artifacts.rake

```diff
@@ -19,11 +19,7 @@ namespace "artifact" do
   SNAPSHOT_BUILD = ENV["RELEASE"] != "1"
   VERSION_QUALIFIER = ENV["VERSION_QUALIFIER"]
   LOCAL_ARTIFACTS = ENV["LOCAL_ARTIFACTS"] || "true"
-  if VERSION_QUALIFIER
-    PACKAGE_SUFFIX = SNAPSHOT_BUILD ? "-#{VERSION_QUALIFIER}-SNAPSHOT" : "-#{VERSION_QUALIFIER}"
-  else
-    PACKAGE_SUFFIX = SNAPSHOT_BUILD ? "-SNAPSHOT" : ""
-  end
+  PACKAGE_SUFFIX = SNAPSHOT_BUILD ? "-SNAPSHOT" : ""
```

Comment:
We no longer need to add the qualifier to the artifact's name explictly here, since it will be added already through the version yml.